letsencrypt/config.yaml

```
@@ -99,8 +100,8 @@ schema:
     propagation_seconds: int(60,3600)?
     provider: "list(dns-azure|dns-cloudflare|dns-cloudns|dns-desec|\
       dns-digitalocean|dns-directadmin|dns-dnsimple|dns-dnsmadeeasy|\
-      dns-duckdns|dns-dynu|dns-gehirn|dns-godaddy|dns-google|\
-      dns-hetzner|dns-infomaniak|dns-joker|dns-linode|dns-luadns|dns-njalla|dns-nsone|\
+      dns-duckdns|dns-dynu|dns-gehirn|dns-godaddy|dns-google|dns-google-domains|\
```

That got dropped with #3750. This probably came in through a merge conflict:

```suggestion
      dns-duckdns|dns-dynu|dns-gehirn|dns-godaddy|dns-google|\
```